layouts/error.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="layout-default">
```

Comment:
@CryptoBender21 
`layout-default` is not available in this component.
Besides that, we should not use global CSS class.
Even we should not use reusable CSS. It's implicit and against component based development like Vue.js, React.js, and Angular.

We should scope CSS and rely on reusable components.

Reply:
Actually this one was an actual mistake. Had the class defined on this scope, removed it and actually forget to remove it on there